LDAP-Auth/LDAPAuthenticationProviderPlugin.cs

```diff
@@ -102,11 +102,17 @@ public async Task<ProviderAuthenticationResult> Authenticate(string username, st
 
                 try
                 {
+                    var adminBaseDn = LdapPlugin.Instance.Configuration.LdapAdminBaseDn;
+                    if (string.IsNullOrEmpty(adminBaseDn))
+                    {
+                        adminBaseDn = ldapUser.Dn;
+                    }
+
                     // Search the current user DN with the adminFilter
                     var ldapUsers = ldapClient.Search(
-                        ldapUser.Dn,
+                        adminBaseDn,
                         LdapConnection.ScopeBase,
-                        AdminFilter,
+                        AdminFilter.Replace(":username", username),
```

Comment:
What is this replace for?

Reply:
sorry, added the description of the feature above.

Reply:
I would prefer `{{username}}` (handlebars mode)
Please also add this to the config page description.